README.md

```diff
@@ -1,5 +1,11 @@
 ![guiltyrobot](guilty-robot.jpg "Guilty Robot")
 
+```
+DEADLINE EXTENDED UNTIL 1st SEPTEMBER
+
+The deadline for 2020 Applications for the Masters in Applied Cybernetics has now been extended until 1st September
```

Comment:
```suggestion
The application deadline for the 2020 Masters in Applied Cybernetics
has now been extended until 1st September.
```

Slight rephrasing, and reorganised for readability on narrow (e.g. mobile) screens.

Reply:
Excellent re-phrasing / re-wrapping suggestion, appreciated. 
Merging :-)